coalib/output/ConsoleInteraction.py

```diff
@@ -766,15 +766,18 @@ def print_bears(bears,
         console_printer.print('No bears to show. Did you forget to install '
                               'the `coala-bears` package? Try `pip3 install '
                               'coala-bears`.')
-        return
+        return None
 
-    for bear, sections in sorted(bears.items(),
-                                 key=lambda bear_tuple: bear_tuple[0].name):
```

Comment:
@Techievena Yes, you are right. but I need to add a unit test for it is well printing by case insensitive. So I need to return this. Do you have better idea to check this test?